MeetingNMusae/MeetingNMusae/Views/BestPlayerShowingView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  BestPlayerShowingView.swift
+//  MeetingNMusae
+//
+//  Created by Heeji Sohn on 2022/06/14.
+//
+
+import SwiftUI
+import FirebaseFirestore
+
+
+//let roomCode = "WP3LX4"
+//let nickname = "Sohni"
+
+
+struct BestPlayerShowingView: View {
+    @State var roomCode: String
```

Comment:
```suggestion
    let roomCode: String
```

roomCode에 값을 변경할 소요가 없어보입니다